app/routers/quiz.py

```
@@ -0,0 +1,74 @@
+            detail={"code": "SERVER_ERROR", "message": "An unexpected error occurred."},
+        )
+
+
+@router.get(
+    "/quizzes/{quiz_id}/questions",
+    response_model=QuizQuestionsResponse,
+    status_code=status.HTTP_200_OK,
+)
+@router.get("/quizzes/{quiz_id}/questions")
+async def get_questions(
+    quiz_id: int, page: int, limit: int, db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)